packages/@okta/vuepress-site/docs/reference/api/itp-et/index.md

```diff
@@ -260,26 +177,101 @@ This resource contains detailed reference material on event types triggered with
-| ID              | For an admin actor, this ID correlates with `user.session.clear` or `user.authentication.universal_logout` events. For a system principal actor, this ID correlates to the `user.authentication.universal_logout` event.               | String         |  `c579b0f27865c4b93be9ceb6f00e5373`          |
-| **target** (User)         | The user associated with a risk activity            | Object     |        |
+| partnerRiskReportData              | The SSF submission from an event provider. It includes the issuer of the security event, the security event URL, and the security event definition.                   | key-value pairs         | `"{  "issuer" : "https://example.eventprovider.com", "https://schemas.openid.net/secevent/caep/event-type/session-revoked" : { "subject" : { "user" : { "format" : "email", "email\" : "joe.alex@example.com" }, "device" : { "format" : "opaque", "sub" : "1234ABCD-123A-123B-123C-12345ABCDEFG" }},    "event_timestamp" : 1709484521, "reason_admin" : {"en" : "Malware detected" }  }}"`         |
+| **target** (User)         |  The user affected by the event           | Object     |        |
+| type        | The type of target object     | String     |       |
+| **actor**                 |  The security events provider                | Object        |         |
+| type        | The type of actor object           | String     | SecurityEventProvider        |
+
+### user.authentication.universal_logout
+
+**Description:** This event triggers when Okta or an admin invokes Universal Logout against an app instance. It contains the app instance details for which the Universal Logout API was triggered. The event indicates when apps have had Universal Logout triggered for audit or debugging purposes. This event is only triggered once. It's only triggered for apps that have been configured for Universal Logout. You can configure it in an Entity risk policy or Continuous Access, or invoke it manually from the user profile.
+
+| Key event properties | Description                                         | Data type      | Example values |
+| --------------------- | --------------------------------------------------- | -------------- | -------------- |
+| **event.system.debugContext.debugData**                |                 |         |         |
+| AppInstanceIds            |A list of application IDs that Okta triggered for Universal Logout                 | Array of IDs         | ["0oa1ysra5y0ESChAr0h8"]        |
+| TraceId            | The `TraceId` is used in continuous access evaluation use cases. A request that triggers a CAE evaluation can ultimately trigger things like CAE action events - and those are executed from the async jobs. `TraceId` connects together events triggered both by the original request handler and from the async jobs triggered by this handler. | String         | `94384405-51e3-4e13-b8b0-ba857b585a63`         |
```

Comment:
We don't use the term "CAE" anywhere

Reply:
Thanks, did a search for all and replaced them.